app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffect.kt

```diff
@@ -33,7 +33,7 @@ object Dismiss : BloodSugarEntryEffect()
 
 object ShowDateEntryScreen : BloodSugarEntryEffect()
 
-data class ShowBloodSugarValidationError(val result: ValidationResult) : BloodSugarEntryEffect()
+data class ShowBloodSugarValidationError(val result: ValidationResult, val unitPreference: BloodSugarUnitPreference) : BloodSugarEntryEffect()
```

Comment:
Can we rename `BloodUnitPreference` to `BloodSugarUnitPreference` in the commit message?

Reply:
Good catch. Fixed it